config/config-autoscaler.yaml

```diff
@@ -54,16 +54,28 @@ data:
     # to achieve efficient resource usage (VPA CPU minimum is 300m).
     container-concurrency-target-default: "100"
 
-    # When operating in a stable mode, the autoscaler operates on the
-    # average concurrency over the stable window.
-    stable-window: "60s"
+    # When observed average concurrency during the panic window reaches
+    # target-panic-percentage the target concurrency, the autoscaler
+    # enters panic mode. When operating in panic mode, the autoscaler
+    # scales on the average concurrency over the panic window which is
+    # window-panic-percentage of the stable-window.
+    window-panic-percentage: "10.0"
 
-    # When observed average concurrency during the panic window reaches 2x
-    # the target concurrency, the autoscaler enters panic mode. When
-    # operating in panic mode, the autoscaler operates on the average
-    # concurrency over the panic window.
+    # Absolute panic window duration.
+    # Deprecated in favor of window-panic-percentage.
+    # Existing revisions will continue to scale based on panic-window
+    # but new revisions will default to window-panic-percentage.
     panic-window: "6s"
 
+    # The percentage of the container concurrency target at which to
+    # enter panic mode when reached within the panic window.
+    target-panic-percentage: "200.0"
```

Comment:
As noted below, this reads to me more like a `panic-concurrency-threshold-factor`. Is there a specific reason to make these `percentage` over a `factor` (which I'd see as percentage/100)

Reply:
(see comment above)